From abaac8dc4ccea63544d0be1ffa7c190e8ae03dd6 Mon Sep 17 00:00:00 2001
From: xubilv <xbl@rock-chips.com>
Date: Wed, 26 Apr 2017 14:47:06 +0800
Subject: [PATCH] video: rockchip: rga2: delay rga2 initcall

rga2 and edp pd is the same -- PD_VIO.
if rga2 initcall earlier than edp,
then it will flash sreen when power on.

Change-Id: Ifa9b4f1f985a6de66d48915f56bc7d225ae0d7a9
Signed-off-by: xubilv <xbl@rock-chips.com>
---
 drivers/video/rockchip/rga2/rga2_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/rga2/rga2_drv.c b/drivers/video/rockchip/rga2/rga2_drv.c
index 2ab9d881e929..a42edc1f6061 100644
--- a/drivers/video/rockchip/rga2/rga2_drv.c
+++ b/drivers/video/rockchip/rga2/rga2_drv.c
@@ -1708,7 +1708,7 @@ void rga2_test_0(void)
 }
 #endif
 
-module_init(rga2_init);
+late_initcall(rga2_init);
 module_exit(rga2_exit);
 
 /* Module information */
-- 
2.35.3

